backend/.env

```diff
@@ -10,4 +10,6 @@ DEFAULT_TL_LANGUAGE=ru
 FEED_INDEX_HOST=feed
 FEED_INDEX_PORT=6767
 SEARCH_HOST=elasticsearch
-SEARCH_PORT=9200
\ No newline at end of file
+SEARCH_PORT=9200
+JWT_ISSUER=
```

Comment:
This is not the right place for these properties.

in `backend/.env` there are only fixed values specific to the backend. Values that are environment-dependent should be in the root `.env`. 

On the other hand, `backend/.env.development` contains all values, needed for the backed in local dev environment (as for local backend run the root .env is not used).

I know we should document this better.

--
TL;DR: move `JWT_ISSUER` and `JWT_ISSUER` from `backend/.env` into `.env.sample`,

but leave them in `backend/.env.development`.